test/test_helper.rb

```diff
@@ -39,7 +39,8 @@ class Application < Rails::Application
       config.secret_key_base = "stub"
       config.eager_load = false
     end
-    ActiveRecord::Base.establish_connection(:adapter => "sqlite3", :database => ":memory:")
+    File.unlink("db/testdb.sqlite3")
```

Comment:
Yeah, also @racerpeter trying to run the specs locally is now blowing up for me with 

```/Users/jeff/workspace/qbwc/test/test_helper.rb:42:in 'unlink': No such file or directory @ unlink_internal - db/testdb.sqlite3 (Errno::ENOENT)```